app/src/main/java/org/jellyfin/mobile/utils/WebAppUtils.kt

```diff
@@ -7,11 +7,14 @@ import java.io.IOException
 
 const val JS_INJECTION_CODE = """
 !function() {
+    window['appMode'] = 'android';
```

Comment:
Might be a matter of taste, but I'd prefer this:
```suggestion
    window.appMode = 'android';
```

Reply:
And I don't

Reply:
Fiiiine, then ignore my request xD

Reply:
A brief search in Google says that the notation by square brackets evaluates what is inside them so it is usually used when the property name is dynamic and/or has special characters but is generally associated with elements of an array, while the dot notation is shorter and is usually associated more with properties of an object, but it has some limitations in the name.

Maybe using the dot notation improves at least 3 yoctoseconds.

Well, it's JavaScript (?)

Reply:
I normally use dot-notation for known properties and square bracket notation for custom properties.